core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1162,6 +1162,10 @@ private[spark] class DAGScheduler(
         partitions = stage.rdd.partitions
       }
 
+      if (taskBinaryBytes.length * 1000 > TaskSetManager.TASK_SIZE_TO_WARN_KB) {
```

Comment:
@liupc, even if it was a minor log change, manual test should better be done to avoid such mistake.

Reply:
@HyukjinKwon Sorry for that, also in the new PR we can make 1000 to 1024, that sounds more reasonable.